test/ered_SUITE.erl

```diff
@@ -256,6 +256,13 @@ t_blackhole(_) ->
 
     ct:pal("Unpausing container: " ++ os:cmd("docker unpause " ++ Pod)),
 
+    %% Since we have {min_replicas, 0} in this testcase, the slot map is not
+    %% updated repeatedly until replicas are found. Instead, we use the
+    %% opportunity to test manual slot update here.
+    ered:update_slots(R),
+    ?MSG(#{msg_type := slot_map_updated}),
```

Comment:
It failed here on the last CI run. I think CLUSTER SLOTS may return the old mapping, because it runs just after the container is unpaused, so maybe the cluster didn't have time to discover it yet.

Probably we need a 2 seconds sleep after unpause.

Reply:
https://github.com/Ericsson/ered/actions/runs/4175329337/jobs/7230093278

Reply:
I added `wait_for_consisten_cluster()` here.